docs/source/models/concept_models/vivarium_mncnh_portfolio/antenatal_care_module/module_document.rst

```diff
@@ -43,7 +43,7 @@ Antenatal care attendance module
 ++++++++++++
 
 This module determines whether or not a simulant attends an antenatal care visit according to their ANC visit 
-propensity value and the timing of their ANC visit (i.e., during first trimester or later pregnancy). 
+probability value and the timing of their ANC visit (i.e., during first trimester or later pregnancy). 
```

Comment:
```suggestion
propensity value and the distribution of ANC attendance exposure categories specific to their location and pregnancy term length. ANC attendance exposure categories include whether a simulant attends an ANC visit during their first trimester and/or later during their pregnancy. 
```
I think this one actually should stay as "propensity"! We use both the propensity and probabilities to determine which exposure to assign to an individual simulant. Let me know if this is unclear and would be helpful to talk through.